### ISSUE TITLE
Error: Callback was already called

### ISSUE THREAD

**Issue by mpirescarvalho** [ytb2mp3/youtube-mp3-downloader#72]
I'm getting this error sometimes:

```javascript
Error: Callback was already called.
    at /app/node_modules/async/dist/async.js:318:36
    at /app/node_modules/youtube-mp3-downloader/lib/YoutubeMp3Downloader.js:39:17
    at FfmpegCommand.<anonymous> (/app/node_modules/youtube-mp3-downloader/lib/YoutubeMp3Downloader.js:161:17)
    at FfmpegCommand.emit (events.js:314:20)
    at emitEnd (/app/node_modules/fluent-ffmpeg/lib/processor.js:426:16)
    at endCB (/app/node_modules/fluent-ffmpeg/lib/processor.js:587:15)
    at handleExit (/app/node_modules/fluent-ffmpeg/lib/processor.js:170:11)
    at ChildProcess.<anonymous> (/app/node_modules/fluent-ffmpeg/lib/processor.js:184:11)
    at ChildProcess.emit (events.js:314:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
```

**Comment by sashilazar** [comment]
I am getting the same error (consistently).  For me it doesn't just happen sometimes.  For a particular video it always happens.

Any chance for a fix?

**Comment by sashilazar** [comment]
Changing lines 161 and 171 from 

      callback(...)

to

     return callback(...)

seems to have fixed the problem for me.

**Comment by mpirescarvalho** [comment]
That didn't work at all for me, still getting the error

**Comment by ytb2mp3** [comment]
Can you please check again with version `0.7.8`?

**Comment by Keristero** [comment]
I'm getting the same error on 0.7.8, in particular with this video
https://www.youtube.com/watch?v=iDVM9KED46Q
It did download successfully, but then my program crashed with the error.

**Comment by mpirescarvalho** [comment]
You have to do what @sashilazar said

replace every 
```
callback(...)
````
with
```
return callback(...)
````

**Comment by tobilg** [comment]
I just released `0.7.9` which should address this. Please check and reopen the issue if it doesn't.
